Source/Core/Core/PowerPC/Interpreter/Interpreter_SystemRegisters.cpp

```diff
@@ -297,6 +297,11 @@ void Interpreter::mtspr(UGeckoInstruction _inst)
     // TODO: emulate locked cache and DMA bits.
     break;
 
+  case SPR_HID4:
+    PowerPC::IBATUpdated();
```

Comment:
What do you think about adding a Jit64 hook for mtspr which emits ResetStack() here?
Else we still have wong blocks within the stack which may be called.